src/Symfony/Bridge/Doctrine/Tests/Validator/Constraints/UniqueEntityValidatorTest.php

```diff
@@ -183,9 +184,7 @@ public function testValidateEntityWithPrivatePropertyAndProxyObject()
         $this->assertNoViolation();
     }
 
-    /**
-     * @group legacy
-     */
+    #[IgnoreDeprecations]
```

Comment:
Unrelated to this PR, but we should update those tests. I'm quite sure most of them are not legacy ones that are supposed to be removed in 8.0 (but they have not been updated to use named arguments for the constraint constructor)